backend/src/controllers/course.controller.ts

```diff
@@ -103,6 +103,31 @@ export class CourseController implements IController {
             return next(err);
           }
         }
+      )
+      .get(
+        "/course/page/:courseCode",
```

Comment:
nit: could you make this endpoint be `/course/:courseCode` instead of `/course/page/:courseCode`